08_cell_line_prediction/run_tcga_ccle_nn.py

```diff
@@ -57,20 +57,45 @@ def process_args():
     p.add_argument('--seed', type=int, default=cfg.default_seed)
     p.add_argument('--shuffle_labels', action='store_true')
     p.add_argument('--verbose', action='store_true')
+
+    # hyperparameter arguments
+    # if these are included they will override the random search ranges
+    # (in pancancer_evaluation/prediction/classification.py), instead providing
+    # a fixed value for the relevant hyperparameter
+    params = p.add_argument_group('params',
```

Comment:
I didn't know you could have a group of arguments like this. Cool!

Reply:
Yeah, it's a cool feature of argparse! It helps if you have a ton of arguments (which is often the case with NNs/complex ML models and their parameters).